lib/streamlit/__init__.py

```diff
@@ -211,8 +211,13 @@ def _update_logger() -> None:
 experimental_user = _UserInfoProxy()
 experimental_singleton = _experimental_singleton
 experimental_memo = _experimental_memo
-experimental_get_query_params = _get_query_params
-experimental_set_query_params = _set_query_params
+# TODO(willhuang1997): Make sure this date is correct. A placeholder is here for now
```

Comment:
TODO's are not great. Will add a todo in the main PR to develop to make sure this gets resolved.